extensions/cep/artifacts/wso2cep-3.0.0/eventformatters/AverageInFlightRequestsWSO2EventFormatter.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2005-2015 WSO2, Inc. (http://wso2.com)
+  ~
```

Comment:
We are no longer using date range for license file.. Please align with latest license headers..